build/build-docker-dev.sh

```diff
@@ -8,6 +8,8 @@ cd "$(dirname $0)/.."
 source ./build/init-docker.sh
 
 cp -p GLOCKFILE build/devbase
+cp -p ui/package.json build/devbase
```

Comment:
i think you want these files in build/devbase/ui (and i think that's why this is failing in CI right now)